core/assets/bundles/bundle.properties

```diff
@@ -1314,4 +1315,26 @@ block.spectre.description = A massive dual-barreled cannon. Shoots large armor-p
 block.meltdown.description = A massive laser cannon. Charges and fires a persistent laser beam at nearby enemies. Requires coolant to operate.
 block.repair-point.description = Continuously heals the closest damaged unit in its vicinity.
 block.segment.description = Damages and destroys incoming projectiles. Laser projectiles are not targeted.
-
+block.resupply-point.description = Refills the ammo supply of units using power.
+block.parallax.description = Fires a tractor beam at air units, making it more difficult for them to fly away.
+block.foreshadow.description = Targets the enemy with the highest health and inflicts massive damage to them.
+block.ground-factory.description = Produces basic ground units.
+block.air-factory.description = Produces basic air units.
+block.naval-factory.description = Produces basic aquatic units.
+block.additive-reconstructor.description = Reconstructs basic units into more advanced units.
+block.multiplicative-reconstructor.description = Reconstructs units from an Additive Reconstructor into more advanced units.
+block.exponential-reconstructor.description = Reconstructs units from a Multiplicative Reconstructor into even more advanced units.
+block.tetrative-reconstructor.description = Reconstructs units from an Exponential Reconstructor into the most advanced units possible.
+block.payload-conveyor.description = Allows for transport of small blocks or units.
+block.payload-router.description = Splits payloads to up to three other directions equally.
+block.disassembler.description = A slag processing structure that can condense thorium from slag. It is less efficient overall, due to requiring raw scrap as well.
+block.silicon-crucible.description = Reduces sand more efficiently into silicon using additional pyratite. Optionally functions faster if placed on the appropriate ground.
+block.overdrive-dome.description = Provides a larger area of overdrive than a standard Overdrive Projector, but requires phase fabric and silicon to operate.
+block.switch.description = A block that can be toggled on or off.
+block.micro-processor.description = A small processor that can perform simple logic instructions. Executes up to 2 instructions in a single logic tick.
```

Comment:
not sure if normal users will understand what a logic tick is, this might need to be translated to per second 🤔